### PR TITLE
Adds more Valentine messages.

### PR DESCRIPTION
**Pull request by harmonyn**
Now that this has been tested and is confirmed not to break everything:
- More valentine/candy heart messages have been added. If the number seems excessive keep in mind the number of people on the server.
- Candy hearts come in a variety of colors.
- Added in a colon I forgot

### CODE REVIEW

**Comment by xxalpha** [comment]
:corn:

**Comment by lordpidey** [comment]
How about:

"Do you have Facial Hypertrichosis?  Because I'd like you to be my beard."

File: code/modules/events/holiday/vday.dm [hunk 1]
```diff
@@ -60,15 +60,27 @@
...
+	               "Do you have stamina damage from running through my dreams?",
+	               "If I were an alien, would you let me hug you?",
+	               "My love for you is stronger than a reinforced wall.",
+	               "This must be the captain's office, because I see a fox.",
+	               "I'm not a highlander, but there can only be one for me.",
+	               "The floor is made of lava! Quick, get on my bed.",
+	               "If you were an abandoned station you'd be the DEARelict.",
+	               "If you had a pickaxe you'd be a shaft FINEr.",
+	               "Roses are red, tide is gray, if I were an assistant I'd steal you away.",
+	               "Roses are red, text is green, I love you more than cleanbots clean.",
```
Comment (KazeEspada) [inline code/modules/events/holiday/vday.dm]: Ooc in ic.

Comment (PKPenguin321) [inline code/modules/events/holiday/vday.dm]: not really

File: code/modules/events/holiday/vday.dm [hunk 2]
```diff
@@ -60,15 +60,27 @@
...
+	               "If you were a carp I'd fi-lay you." )
 
 /obj/item/weapon/valentine/attackby(obj/item/weapon/W, mob/user, params)
 	if(istype(W, /obj/item/weapon/pen) || istype(W, /obj/item/toy/crayon))
 		var/recipient = stripped_input(user, "Who is receiving this valentine?", "To:", null , 20)
 		var/sender = stripped_input(user, "Who is sending this valentine?", "From:", null , 20)
 		if(recipient && sender)
-			name = "valentine - To: [recipient] From [sender]"
+			name = "valentine - To: [recipient] From: [sender]"
```
Comment (Jalleo) [inline code/modules/events/holiday/vday.dm]: Colon for consistency in grammar :+1:

File: code/modules/events/holiday/vday.dm (hunk 1 above)
Comment (bgobandit) [inline code/modules/events/holiday/vday.dm]: there's already a greentext item

File: code/modules/events/holiday/vday.dm [hunk 3]
```diff
@@ -60,15 +60,27 @@
 	               "If I were the warden I'd always let you into my armory.",
 	               "The virologist is rogue, and the only cure is a kiss from you.",
 	               "Would you spend some time in my upgraded sleeper?",
-	               "You must be a silicon, because you've unbolted my heart."
-	               "Are you Nar-Sie? Because there's nar-one else I sie." )
+	               "You must be a silicon, because you've unbolted my heart.",
+	               "Are you Nar-Sie? Because there's nar-one else I sie.",
+	               "If you were a taser, you'd be set to stunning.",
```
Comment (Chocobro) [inline code/modules/events/holiday/vday.dm]: i'm laughing a fucking lot holy shit i will use this and get immediately slapped

**Pull request merged**